src/App.tsx

```diff
@@ -8,7 +8,61 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { FilterBy, Todo } from './types/Todo';
+import { User } from './types/User';
+import { getTodos, getUser } from './api';
+import { filterTodos } from './service/todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setloading] = useState(false);
```

Comment:
we usually name boolean vars starting with `is`
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```